client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -80,10 +80,10 @@ export default function Dashboard() {
         <Featured />
       </div>
       <div className="sessionsContainer">
-        <Card sessions={sessions} name={'All'} />
+        <Card sessions={sessions} name={'All Sessions'} />
         <Card sessions={nearYouSessions} name={'Activities near you'} />
         {interestedSportsSessions.map((sessionsArray, index) => (
-          sessionsArray && sessionsArray[0] && <Card key={index} sessions={sessionsArray} name={sessionsArray[0].sport} />
+          sessionsArray && sessionsArray[0] && <Card sessions={sessionsArray} name={sessionsArray[0].sport} />
```

Comment:
I think we should not remove the key, otherwise I get this error
<img width="648" alt="Screenshot 2023-08-08 at 10 58 50 PM" src="https://github.com/srujanr40/dogfish/assets/13267569/2e635881-3a7e-49dc-9c89-6464b4daca23">

Reply:
Done.